core/src/main/scala/sttp/client3/SttpBackendOptions.scala

```diff
@@ -36,7 +36,8 @@ object SttpBackendOptions {
       port: Int,
       proxyType: ProxyType,
       nonProxyHosts: List[String] = Nil,
-      auth: Option[ProxyAuth] = None
+      auth: Option[ProxyAuth] = None,
+      hostsToProxy: List[String] = Nil
```

Comment:
maybe to keep naming symmetry this should be called e.g. `useProxyHosts`?

Reply:
I don't think that `useProxyHosts` reflects that _only_ those hosts will be proxied. 

How about `onlyProxyHosts`? It keeps to the symmetry and better (I think) shows what it does

Reply:
yeah this looks good. I'll do the rename quickly, thanks for the suggestion :)